# Multi-omic Data in Cervical Cancer Studies using Machine Learning

[![License](https://img.shields.io/badge/license-MIT-blue.svg)](LICENSE)

This repository contains code and resources for the research project titled "Multi-omic Data in Cervical Cancer Studies using Machine Learning."

## Aim of this research

- Classify cervical cancer into its subgroups using machine learning model.

## Dataset

We utilize the TCGA (The Cancer Genome Atlas) data for our research. The following omics are used:

- miRNA Genome Data
- Gene Expression Profiling Data
- DNA Methylation Data

Please note that due to data privacy restrictions, the TCGA dataset cannot be included in this repository. However, we provide instructions on how to access and preprocess the data in the `data_preparation` directory.

## Repository Structure

The repository is organized as follows:

- `data_preparation/`: Code and instructions for accessing and preprocessing the TCGA data.
- `feature_selection/`: Code for feature selection and analysis.
- `model_training/`: Code for training machine learning models using multi-omic data.
- `evaluation/`: Code for evaluating the performance of the models.
- `results/`: Directory to store the results and outputs generated during the research.
- `references/`: Relevant papers and articles used for literature review.

## Requirements

To run the code in this repository, you will need the following dependencies:

- Python 3.x
- NumPy
- Pandas
- Scikit-learn
- Matplotlib
- Seaborn

Please refer to the individual directories for specific requirements and installation instructions.

## Contributing

Contributions to this repository are welcome. If you find any issues or have suggestions for improvements, please open an issue or submit a pull request.

## License

This project is licensed under the MIT License. See the [LICENSE](LICENSE) file for details.

